kolla-ansible/ansible/triliovault_globals.yml

```diff
@@ -84,15 +84,17 @@ triliovault_s3_version: 'default'
 ## S3 Auth version
 triliovault_s3_auth_version: 'DEFAULT'
 
-## Valid for 'other_s3_compatible' backup target only, If SSL is enabled on S3 endpoint url then change it to 'True', otherwise keep it as 'False'
+## Valid for 'other_s3_compatible' backup target only, if endpoints on 'https' ie SSL Enabled 
+## then change it to 'True', otherwise keep it as 'False'
 triliovault_s3_ssl_enabled: False
-## Valid for 'ceph_s3' backup target only, if SSL is enabled on S3 endpoint URL and SSL certificates are self signed
-# OR issued by a private authority then, user needs to copy the 'ceph s3 ca chain file' to "/etc/kolla/config/triliovault/" 
-# directory on ansible server. You need to create this directory if not exists already.
-# And provide the cert file name as value of following parameter.
+
+## Valid for 'other_s3_compatible' backup target only, if endpoints on 'https' ie SSL Enabled
+## then user needs to copy the 'ceph s3 ca chain file' to "/etc/kolla/config/triliovault/" 
+## directory on ansible server. You need to create this directory if not exists already.
+## And provide the cert file name as value of following parameter.
 triliovault_s3_ssl_cert_file_name: 's3-cert.pem'
 
-## Valid for 'ceph_s3' backup target only, if SSL is enabled on S3 endpoint URL and SSL certificates are self signed or issued by private authority,
+## Valid for 'other_s3_compatible' backup target only, if endpoints on 'https' ie SSL Enabled
 ## then, user needs to set this parameter value to: True , otherwise keep it's value as False.
 ## Type: Boolean
 triliovault_copy_ceph_s3_ssl_cert: False
```

Comment:
do we need to change this parameter name triliovault_copy_ceph_s3_ssl_cert ? It is referring ceph-s3 here. You also need to check at the backend if we changing this paarm name then it should not break the code when used with/without ceph s3 .